apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs

```diff
@@ -287,7 +287,9 @@
                       <i class="fa fa-user-plus" aria-hidden="true"></i> Add members
                     </a>
 
-                    <a class="dropdown-item" onclick="$('#row<%= iRow %>Popoverdeletemember').popover('show')" href="#">
+                    <a class="dropdown-item<%= row.users.length === 0 ? ' disabled' : '' %>"
+                       <% if (row.users.length > 0) { %>onclick="$('#row<%= iRow %>Popoverdeletemember').popover('show')"<% } %>
+                       href="#">
```

Comment:
I hadn't actually looked at the markup for this before. Could we use a button here instead? That would let us avoid some of the complexity with conditionals.

Reply:
I only saw this after it was already merged. Do you want me to address that in a separate PR?

Reply:
If you wouldn't mind, that'd be great.